src/styles/_carousel.scss

```diff
@@ -143,7 +143,7 @@
 		}
 	}
 
-	&__link,
+	&__Link,
```

Comment:
use small letter for the css. Thanks.

Reply:
this has been address with latest commit